cmd/operator-sdk/main.go

```diff
@@ -81,3 +84,77 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func checkDepManagerForCmd(cmd *cobra.Command) (err error) {
+	// Certain commands are able to be run anywhere or handle this check
+	// differently in their CLI code.
+	if skipCheckForCmd(cmd) {
+		return nil
+	}
+	// Do not perform this check if the project is non-Go, as thy will not have
```

Comment:
```suggestion
	// Do not perform this check if the project is non-Go, as they will not have
```
Unless you meant to use Ye Olde English.